### PR TITLE
Feat ambassadors

### PR DESCRIPTION
**Pull request by JakeatRipple**

### CODE REVIEW

File: assets/css/devportal2022-v4.css [hunk 1]
```diff
@@ -3,4 +3,4 @@
...
....done a::after{color:#145C35}html.light .modal-content{background-color:#FCFCFD}html.light .rpc-tool pre .toggle{color:#fff}html.light .rpc-tool pre .toggle:hover{color:#B480FF}html.light .page-home #home-hero-graphic{content:url("../img/lightmode/home-hero.svg")}html.light .page-home #benefits-list #public{content:url("../img/icons/lightmode/public.svg")}html.light .page-home #benefits-list #streamlined{content:url("../img/icons/lightmode/streamlined.svg")}html.light .page-home #benefits-list #performance{content:url("../img/icons/lightmode/performance.svg")}html.light .page-home #benefits-list #low-cost{content:url("../img/icons/lightmode/low-cost.svg")}html.light .page-home #benefits-list #community{content:url("../img/icons/lightmode/community.svg")}html.light .page-home #benefits-list #reliability{content:url("../img/icons/lightmode/reliability.svg")}html.light #validator-graphic{content:url("../img/lightmode/validators.svg")}html.light #wallets #wallet-xumm{content:url("../img/wallets/lightmode/xumm.svg")}html.light #top-exchanges #exch-bitstamp{content:url("../img/exchanges/lightmode/bitstamp.svg")}html.light #top-exchanges #exch-cex-io{content:url("../img/exchanges/lightmod
```
Comment (mDuo13) [inline assets/css/devportal2022-v4.css:6]: Please remove unused old versions of the CSS file.

File: template/page-ambassadors.html.jinja [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+                            <p class="">Read more about the activity of the XRPL Ambassadors</p>
+                        </div>
+                    </div>
+
+                </div>
+
+                <div class="col-12 col-lg-6 p-0 pl-lg-4">
+                    
+                    <div class="p-lg-3 mb-3 pb-3 ">
+                        <img src="./assets/img/ambassadors/icon_discord.svg" class="mb-3">
```
Comment (mDuo13) [inline template/page-ambassadors.html.jinja:485]: Like on the Community page, the blue Discord logo is low-contrast on the dark theme. Can we get a light/dark toggle for this icon?

**Review by mDuo13 — CHANGES_REQUESTED** [review]
Looks very good. Just a couple fixes I'd like to see before we publish it.

File: assets/css/devportal2022-v4.css (hunk 1 above)
Comment (JakeatRipple) [reply assets/css/devportal2022-v4.css:6]: Removed.

File: template/page-ambassadors.html.jinja (hunk 2 above)
Comment (JakeatRipple) [reply template/page-ambassadors.html.jinja:485]: Updated the color icon and also added white version to toggle.

**Review by mDuo13 — APPROVED** [review]

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**